src/olympia/amo/utils.py

```diff
@@ -782,6 +782,8 @@ def rm_local_tmp_dir(path):
     certain that your executing code is operating on a local temp dir, not a
     directory managed by the Django Storage API.
     """
+    assert path.startswith(settings.TMP_PATH)
+
     return shutil.rmtree(path)
```

Comment:
This needs special attention since I almost puked reading the comment and seeing that we don't ensure paths match the functions expectation. Not sure that assertion is needed but I feel like it might let me sleep better. Opinions?

Reply:
+1 on the assert.